src/foam/nanos/notification/email/DAOResourceLoader.java

```diff
@@ -42,6 +43,7 @@ public static EmailTemplate findTemplate(X x, String name, String groupId, Strin
     DAO groupDAO = (DAO) x.get("groupDAO");
     DAO emailTemplateDAO = (DAO) x.get("localEmailTemplateDAO");
     EmailTemplate emailTemplate = null;
+    Logger logger = (Logger) x.get("logger");
```

Comment:
```suggestion
    Logger logger = Loggers.logger(x, this);
```
This will create a logger that will prefix log messages this class name so we know where they originated.